### PR TITLE
Use emscripten 3.1.13

### PR DESCRIPTION
**Pull request by hoodmane**
On top of #2672.

### CODE REVIEW

File: packages/scipy/patches/0001-Fix-dstevr-in-special-lapack_defs.h.patch [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+From 9aeab0fdc80d21b5a9a88402fdcbdf4c6e0589bc Mon Sep 17 00:00:00 2001
+From: Hood Chatham <roberthoodchatham@gmail.com>
+Date: Fri, 18 Mar 2022 16:25:39 -0700
+Subject: [PATCH] Fix dstevr in special/lapack_defs.h
+=======
 From 90a2cac7e8812f0a3e64c6bbc45fb1ece8d5f4d1 Mon Sep 17 00:00:00 2001
 From: Hood Chatham <roberthoodchatham@gmail.com>
 Date: Fri, 18 Mar 2022 16:25:39 -0700
 Subject: [PATCH 01/14] Fix dstevr in special/lapack_defs.h
+>>>>>>> main
```
Comment (rth) [inline packages/scipy/patches/0001-Fix-dstevr-in-special-lapack_defs.h.patch:11]: A left over merge conflict

File: packages/scipy/patches/fix-implicit-cast-args-from-newer-lapack.patch [hunk 2]
```diff
@@ -0,0 +1,23 @@
+We are hijacking dfft.f to inject missing functions from future versions of LAPACK.
+
+This patch is applied to the generated dfft.c after f2c is applied to dfft.f. 
+This must be done in between when f2c is applied to dfft.f and when dfft.c is compiled.
+This is arranged in _f2c_fixes.py.
```
Comment (rth) [inline packages/scipy/patches/fix-implicit-cast-args-from-newer-lapack.patch:5]: Sigh, I'm eagerly waiting for lfortran to be able to compile scipy for wasm.

**Review by rth — APPROVED** [review]
Thanks @hoodmane !

File: packages/scipy/patches/fix-implicit-cast-args-from-newer-lapack.patch (hunk 2 above)
Comment (hoodmane) [reply packages/scipy/patches/fix-implicit-cast-args-from-newer-lapack.patch:5]: I think this is also caused by bad merge. This patch isn't used.

**Comment by hoodmane** [comment]
And we are on the most recent Emscripten version for the first time in over a year.

**Comment by rth** [comment]
Great work, @hoodmane !

**Pull request merged**